code/src/cljs/sixsq/nuvla/ui/utils/general.cljs

```diff
@@ -94,6 +94,11 @@
        str/join))
 
 
+(defn capitalize-first-letter
```

Comment:
Is there a reason to redefine str/capitalize ?

Reply:
It’s unfortunately not exactly the same. This function only capitalizes the first letter, leaving the rest of the word unchanged. This allows NuvlaBox to keep the upper case ‘b’.

Reply:
Thanks for the explanation 👍